src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -149,22 +149,19 @@ void tryCompileDeepFluentCalls(int depth)
             }
         }
 
-        [Fact]
+        [ConditionalFact(typeof(WindowsOrLinuxOnly))]
```

Comment:
Limited the execution architectures to reduce the number of magic numbers that need to be maintained below. The test should still catch regressions where we change the stack overhead for deeply-nested generic types with the remaining execution architectures.

Reply:
Was this primarily because of the additional effort needed to tune the magic numbers on different platforms?

Reply:
> Was this primarily because of the additional effort needed to tune the magic numbers on different platforms?

Yes. We should get sufficient coverage from the remaining platforms, so including these platforms seemed unnecessary effort.